tests/unit/test_config.py

```diff
@@ -396,7 +396,7 @@ def test_process_icp(
         assert mocked_config.cfg[
             'scheduling']['initial cycle point'] == expected_icp
         assert str(mocked_config.initial_point) == expected_icp
-        opt_icp = mocked_config.options.icp
+        opt_icp = mocked_config.evaluated_icp
         if opt_icp is not None:
             opt_icp = str(loader.get_point(opt_icp).standardise())
         assert opt_icp == expected_opt_icp
```

Comment:
Can you replace `opt` with `evaluated` in references to `opt_icp` and `expected_opt_icp`, otherwise this no longer makes sense?